src/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -82,22 +82,42 @@ void KickoffEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield,
         std::make_shared<MoveTactic>(true), std::make_shared<MoveTactic>(true),
         std::make_shared<MoveTactic>(true)};
 
+    // created a new local variable Point at (zero,zero)
+    Point point = Point();
+    // created an enemy_team for mutation
+    Team enemy_team = world.enemyTeam();
+
     do
     {
         // TODO: (Mathew): Minor instability with defenders and goalie when the ball and
         // attacker are in the middle of the net
+
+        // We find the nearest enemy robot closest to (0,0) then ignore it from the enemy
+        // team
+        if (Team::getNearestRobot(world.enemyTeam().getAllRobots(), point).has_value())
```

Comment:
nit: would be good to call this once, store the result, and then check on that

`auto robot = Team::getNearestRobot(world.enemyTeam().getAllRobots(), point)`

Reply:
Done